src/App.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Why did this file get added back?

Reply:
I synced my fork and then it must have added it back but I am personally not using it.

Reply:
Can you please remove that file from your commit then?

Reply:
Removed file